frontend/src/pages/ApprovedActivityReport/index.js

```diff
@@ -94,7 +94,7 @@ function mapAttachments(attachments) {
         {
           attachments.map((attachment) => (
             <li key={attachment.url.url}>
-              <a href={attachment.url.url}>{attachment.originalFileName}</a>
+              <a href={attachment.url.url} target={attachment.originalFileName.endsWith('.txt') ? '_blank' : '_self'} rel="noreferrer">{attachment.originalFileName}</a>
```

Comment:
Why are we handling text files different from every other file?

Reply:
The reason we are doing this is because the text file will always open in the browser (current tab). My work around to this was to get it to open in a new tab.

Reply:
One solution was to set the a href property 'download' to true but this didn't work.

Reply:
Cool, was just wondering!